src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,24 @@
+.catalog {
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
```

Comment:
it is not necessary to add justify-content

Reply:
Justify-content: center; was added to center cards in container, even if we delete one of them.

Reply:
No, we don't want them to be in the center of the container.
If I delete one, left cards should stay in the same position